modules/ocl/cl_context.cpp

```diff
@@ -423,10 +434,15 @@ CLContext::import_dma_buffer (const cl_import_buffer_info_intel &import_info)
 {
     cl_mem mem_id = NULL;
     cl_int errcode = CL_SUCCESS;
-    if (!is_valid())
+    cl_platform_id platform_id = _device->get_platform_id ();
+    if (!is_valid() || !platform_id)
         return NULL;
 
-    mem_id = clCreateBufferFromFdINTEL (_context_id, &import_info, &errcode);
+    OCLCREATEBUFFERFROMFDINTEL *oclCreateBufferFromFdINTEL =
+        (OCLCREATEBUFFERFROMFDINTEL *) clGetExtensionFunctionAddressForPlatform (platform_id, "clCreateBufferFromFdINTEL");
```

Comment:
move clGetExtensionFunctionAddressForPlatform  to cl_device.cpp.
add interface void *CLDevice::get_extension_function(const char* func_name);